.appveyor.yml

```diff
@@ -15,41 +15,46 @@ environment:
         #    PyQt5 only available for Python >=3.5.
         #       PyQt5 version 5.11.2 doesn't run on Python 3.5.
         #    PySide1 only available for 2.6 <= Python <=3.4. Wheels only for 32-bit.
-        
+
         # Strategy:
+        #    Run all tests using only pip (Travis uses conda)
         #    Test each 64-bit Python version, but only the latest known patch.
-        #    Test the oldest and the latest 32-bit Python versions release. 
-        #    Test with PyQt4 where available, otherwise PyQT5 in each version.
+        #    Test the oldest and the latest 32-bit Python versions release.
+        #    Test with PyQt4 where available, otherwise PyQt5 in each version.
         #    Test QScintilla on latest Python version available.
         #    Test Pyside1 in latest Python version it runs on.
         #        - Failure is allowed and won't block PRs.
         #    Test Pyside2 in latest Python version available
         #        - Failure is allowed and won't block PRs.
         #    Only test single toolkit installs (e.g. not QT4 and QT5 on same install)
-        
+
         # VARIABLES
         #    NAME: just a comment for distinguishing builds in Appveyor.
         #    PYQT_VERSION: Determines PyQt versions installation
-        #    PY_QT: Determines Pyside version installation (if present), and is used by 
+        #    PY_QT: Determines Pyside version installation (if present), and is used by
         #           Enaml.
         #    PYTEST_QT_API: In Python 3.4, pytest-qt needs a hint about the right interface.
         #    QSCINTILLA: Set to "Yes" to install it.
-        
+
 
         # 64-bit versions, PyQT, latest to oldest.
-        
+
+        - NAME: "Py 3.7"
+          PYTHON: "C:\\Python37-x64"
+          PYTHON_VERSION: "3.7.0"
+          PYTHON_ARCH: "64"
```

Comment:
It would shorten the build for sure. But there is no pain in maintaining them so I will probably keep them but I will group QScintilla with latest. The main issue is getting Appveyor not to build twice every commit in PRs.

Reply:
Sorry, I seem to have been unclear.

I am all for testing on 32 bit and 64 bit architectures.

However, the environment variable PYTHON_ARCH doesn't contribute much. It is only used for some debug output. The actual selection of the architecture is done by the path to Python, not the PYTHON_ARCH variable.